media/css/m24/components/navigation-refresh.scss

```diff
@@ -141,6 +141,10 @@ $margin-top: 54px; // top margin offset for mobile navigation menu
     }
 }
 
+[dir='rtl'].js .m24-c-navigation-menu-button {
```

Comment:
This was necessary because the direction is applied to the same element that gets the .js class, so the bidi() mixin wasn't really working. It could be fixed more properly with some further refactoring of the CSS but this was a quick hack.